react/QtdSelector.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+
+/**
+* The quantity selector component.
+* It constrols how many products a client wants to buy.
+*/
+class QtdSelector extends Component {
+  constructor(props) {
+    super(props)
+    this.handleChange = this.handleChange.bind(this)
+  }
+
+  handleChange(event) {
```

Comment:
Bind the function using an arrow binding here.
Ask @estacioneto about it.

Reply:
Done!